test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreRetryTests.cs

```diff
@@ -98,7 +98,7 @@ public void RestoreRetry_PackagesConfigRetryOnFailingV2Source()
 
                         throw new Exception("This test needs to be updated to support: " + path);
                     });
-
+                    pathContext.Settings.AddSource("http-feed", $"{server.Uri}nuget", allowInsecureConnectionsValue: "true");
```

Comment:
Does this replace a feed? 
I'm not sure I understand why we need it.

Reply:
The mock server is http. And as a result all these restore test cases would fail. Adding it to the config file with the `allowInsecureConnections` atttribute enabled, allows for the nuget.exe to be executed without failing.

Reply:
I get that, but is it replacing a feed?
The reason I'm asking is because I didn't see this before in the test.

Reply:
Talked offline:

The feed wasn't specified before, but we're specifying it now to allow the -source option to work without warning.